cdc/generation.cc

```diff
@@ -302,13 +302,13 @@ db_clock::time_point make_new_cdc_generation(
         const gms::gossiper& g,
         db::system_distributed_keyspace& sys_dist_ks,
         std::chrono::milliseconds ring_delay,
-        bool for_testing) {
+        bool add_delay) {
```

Comment:
nitpick - the name is somewhat misleading, because even if add_delay=true we *don't* add the delay when ring_delay=0 (there is a "||" below). But I guess we can live with that.

Reply:
yeah. I thought about it when I was changing the name and couldn't have figured a better way.
One solution would be to move the ring_delay == 0 check out of the method.